src/main/resources/data.sql.dumy

```diff
@@ -0,0 +1,12 @@
+-- FYI: any camelCase field name becomes camel_case in sql land
+
+INSERT INTO course(id, name, quarter, instructor_first_name, instructor_last_name, instructor_email) VALUES ('1', 'CMPSC 156', '20204', 'Phill', 'Conrad', 'phtcon@ucsb.edu');
+INSERT INTO course(id, name, quarter, instructor_first_name, instructor_last_name, instructor_email) VALUES ('2', 'CMPSC 148', '20204', 'Chandra', 'Krintz', 'ckrintz@ucsb.edu');
+
+INSERT INTO tutor(id, first_name, last_name, email) VALUES ('1', 'Andrew', 'Lu', 'alu@ucsb.edu');
+INSERT INTO tutor(id, first_name, last_name, email) VALUES ('2', 'Scott', 'Chow', 'scottpchow@ucsb.edu');
+
+INSERT INTO tutor_assignment(id, tutor_id, course_id, assignment_type) VALUES ('1', '2', '1', 'TA');
+INSERT INTO tutor_assignment(id, tutor_id, course_id, assignment_type) VALUES ('2', '1', '1', 'LA');
+
+INSERT INTO online_office_hours(id, tutor_assignment_id, day_of_week, start_time, end_time, zoom_room_link, notes) VALUES ('1', '1', 'Tuesday', '4:00 PM', '6:00 PM', 'https://ucsb.zoom.us/j/92714889391?pwd=dXlaVTFUS1NuNm5xL0NMUUFjRDVndz09', 'Scott closes the room early sometimes but he will still be on slack!');
```

Comment:
We already have a sample `data.sql` file, can we remove this "dumy" file?

Reply:
The file is removed